### DISCUSSION TITLE
docker build . -t log4shell command fails

### DISCUSSION THREAD

**Discussion by ajithshetty** [lunasec-io/lunasec#426] [Help/Support] [answered]
Im trying to build using the Dockerfile and I see this error:

#16 65.60 # github.com/lunasec-io/lunasec/tools/log4shell
#16 65.60 ./main.go:26:27: undefined: cli
#16 65.60 ./main.go:62:32: undefined: cli
#16 65.60 ./main.go:71:10: undefined: cli
------
executor failed running [/bin/sh -c CGO_ENABLED=0 go build -o log4shell .]: exit code: 2

Trying out in MAC Any suggestions?

**Comment by freeqaz** [comment, marked as answer]
I just landed a fix for this in #427 to master. Thanks for bringing this up!

**Reply by ajithshetty** [reply]
Thank you @freeqaz
